ansible/roles/desktop-deploy/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Copy the desktop assests folder from inside the private repo location to root location.
+  shell: "cp -r {{offline_repo_location}}/private/desktop-assets {{offline_repo_location}}/"
```

Comment:
Would suggest to use the copy module here

Reply:
copy module doesn't work internal to the server.

Reply:
I disagree on this. Copy can work local as well as remote. But I am ok with using a shell here.